Pages/GmailPage.py

```diff
@@ -0,0 +1,113 @@
+import csv
+import json
+import os
+
+from datetime import date
+from Constans import uri, element_tuples, arrays
```

Comment:
na arba sudeti i viena faila arba del tvarkingumo rasyti *, nes nezinau ar daug memory susitaupo neinportinant failo su viena eilute, na bet cia paciam idomu suzinoti kokia geresne praktika yra

Reply:
from Constans import * neveikia ir kad kazka panasaus prasukt reikia papildomo kodo. Sprendziant is cia: https://stackoverflow.com/questions/1057431/how-to-load-all-modules-in-a-folder kaip supratau dauguma linksta prie to kad tai yra bad praktice. Correct me if Im wrong